src/roman-numerals/index.js

```diff
@@ -4,6 +4,34 @@
  * @param {string} roman The all-caps Roman numeral between 1 and 3999 (inclusive).
  * @returns {number} The decimal equivalent.
  */
-function romanToDecimal(roman) {}
+function romanToDecimal(roman) {let  mainStr = roman.toUpperCase()
```

Comment:
You really should format your code.

Makes it easier to read.

Reply:
Noted... the other roman numerals file in the /brackets was a mistake... i'll delete it.